packages/language-server/test/langauge-services.spec.js

```diff
@@ -27,10 +24,12 @@ describe("the XML Language Services", () => {
           end: pos,
         },
         severity: DiagnosticSeverity.Error,
-        source: SYNTAX_ERROR_MSG,
+        source: "XML LS",
       },
     ];
-    const diagnostics = await validateDocument(doc);
+    const diagnostics = await validateDocument(doc, {
+      consumer: DEFAULT_CONSUMER_NAME,
```

Comment:
Maybe send the value hard-coded here and make it different from the actual default value in at least one test? Since you want to check that this is the source you get back. If the code always returned the default consumer name this won't validate that you get what you sent in the consumer.

Reply:
good idea

Reply:
done in: https://github.com/SAP/xml-tools/pull/297/commits/97b4a7aca8c13826c5118af7b5b8bb607c7adaee